libbeat/publisher/pipeline/module.go

```diff
@@ -80,8 +80,8 @@ func Load(
 		Annotations: Annotations{
 			Event: config.EventMetadata,
 			Builtin: common.MapStr{
-				"beat": common.MapStr{
-					"name":     name,
+				"agent": common.MapStr{
```

Comment:
@graphaelli I think we should must make it possible for the apm-server to overwrite these fields with the agent information. Meaning we should only set them if they are not already in the event.

Reply:
@graphaelli Will make this happen in a follow up PR. Marked it in the ECS tracking issue: https://github.com/elastic/beats/issues/8655